firefox-ios/Client/Frontend/Home/Homepage Rebuild/HomepageViewController.swift

```diff
@@ -87,6 +90,11 @@ final class HomepageViewController: UIViewController,
         applyTheme()
     }
 
+    override func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
+        super.viewWillTransition(to: size, with: coordinator)
+        wallpaperView.updateImageForOrientationChange()
```

Comment:
[This comment ](https://github.com/mozilla-mobile/firefox-ios/pull/23202#discussion_r1847349929) did make me wonder if orientation changes need to be redux actions as well or if the view controller can just directly update the view.

Reply:
It seems based on [this discussion](https://mozilla.slack.com/archives/C05C9RET70F/p1732032383378539), It does not need to be part of redux:
>  Broadcasting an action when you know there will be nobody listing for it is not necessary. Someone can always come back later and add it if a use case pops up where they need to listen for it.